actions/setup_jdk/action.yml

```diff
@@ -8,20 +8,63 @@ inputs:
   java-version:
     description: 'Java version'
     required: false
-    default: '19.0.2'
+    default: '20.0.2'
   distribution:
     description: 'Distribution (e.g. "adopt")'
     required: false
-    default: 'adopt' # Adopt OpenJDK
+    default: 'custom' # Adopt OpenJDK
+  downloadUrl:
+    description: 'URL to dwonload JDK archive. Should be used if `distribution` is custom.'
+    required: false
+    default: "https://download.java.net/java/GA/jdk20.0.2/6e380f22cbe7469fa75fb448bd903d8e/9/GPL/openjdk-20.0.2_linux-x64_bin.tar.gz"
```

Comment:
We can skip checksum at all, but URL can't be generated anyways, because it has `6e380f22cbe7469fa75fb448bd903d8e` this part in it. I guess it is git commit, or some hash. It can't be inferred from version. 

And we can't use JDKs available to base actions, as we want actual official OpenJDK.

I don't think providing mapping from version to URL is viable option. It would force us to just maintain list of versions somewhere. I think if caller really need some specific version -- they can provide URL, and by default we don't provide url to OUR action, so it uses default value.


Should we remove checksum at all?

Reply:
As discussed, let's keep it as is for now